spring-cloud-tencent-polaris-context/src/main/java/com/tencent/cloud/polaris/context/PolarisContextAutoConfiguration.java

```diff
@@ -36,8 +39,10 @@ public class PolarisContextAutoConfiguration {
 
 	@Bean(name = "polarisContext", initMethod = "init", destroyMethod = "destroy")
 	@ConditionalOnMissingBean
-	public SDKContext polarisContext(PolarisContextProperties properties) throws PolarisException {
-		return SDKContext.initContextByConfig(properties.configuration());
+	public SDKContext polarisContext(PolarisContextProperties properties,
```

Comment:
After spring boot 2.6.x version ,`allowCircularReferences` default value is `false` . 

```java
/**
  * Sets whether to allow circular references between beans and automatically try to
  * resolve them. Defaults to {@code false}.
  * @param allowCircularReferences if circular references are allowed
  * @since 2.6.0
  * @see AbstractAutowireCapableBeanFactory#setAllowCircularReferences(boolean)
  */
public void setAllowCircularReferences(boolean allowCircularReferences) {
    this.allowCircularReferences = allowCircularReferences;
}
```

```properties
spring.main.allow-circular-references=true
```

So , here is fixed circular references .

Reply:
@SkyeBeFreeman

Reply:
LGTM.